target/dovecot/10-mailcrypt.disabled

```diff
@@ -0,0 +1,7 @@
+mail_attribute_dict = file:%h/.attributes
+mail_plugins = $mail_plugins mail_crypt
+plugin {
+  mail_crypt_curve = secp521r1
```

Comment:
There isn't much from Google beyond the Dovecot docs which use this curve. I don't mind it personally, it's not likely to be a performance concern or anything.

It may affect some users that need to use only government approved curves though? [This stackexchange answer expands on that](https://security.stackexchange.com/questions/100991/why-is-secp521r1-no-longer-supported-in-chrome-others), [Suite B has been replaced by CNSA since 2018 however](https://en.wikipedia.org/wiki/NSA_Suite_B_Cryptography#Commercial_National_Security_Algorithm_Suite), [CNSA advises P-384 (secp384r1)](https://apps.nsa.gov/iaarchive/programs/iad-initiatives/cnsa-suite.cfm).

Reply:
I've added `PER_USER_STORAGE_ENCRYPTION_CURVE` to solve this